src/pages/Dashboard.tsx

```diff
@@ -1,14 +1,17 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import NoOrganization from '../components/NoOrganization';
 import organizationService from '../services/organization.service';
 import { getPreference } from '../services/preference.service';
 
 function Dashboard() {
   const history = useHistory();
+  const [organizationCount, setOrganizationCount] = useState(-1);
```

Comment:
Could you initialize it like this? The -1 feels a little weird.

```suggestion
  const [organizationCount, setOrganizationCount] = useState<number>();
```